CSharp/DevVmPowershell/DevVmPsModules/DocumentModule.cs

```diff
@@ -116,10 +116,10 @@ private void ValidateInputArguments()
 				throw new ArgumentNullException(nameof(FileCount), $"{nameof(FileCount)} cannot be less than 0.");
 			}
 
-			if (string.IsNullOrWhiteSpace(ResourceFilePath))
-			{
-				throw new ArgumentNullException(nameof(ResourceFilePath), $"{nameof(ResourceFilePath)} cannot be NULL or Empty.");
-			}
+			//if (string.IsNullOrWhiteSpace(ResourceFilePath))

```

Comment:
can we remove the comments if not needed?

Reply:
Updated in last commit